src/Owin.Security.Providers.Steam/SteamAuthenticationHandler.cs

```diff
@@ -14,11 +14,14 @@ internal sealed class SteamAuthenticationHandler : OpenIDAuthenticationHandlerBa
 
         private const string UserInfoUri = "http://api.steampowered.com/ISteamUser/GetPlayerSummaries/v0002/?key={0}&steamids={1}";
 
+        private const string XmlSchemaString = "http://www.w3.org/2001/XMLSchema#string";
+
         public SteamAuthenticationHandler(HttpClient httpClient, ILogger logger) : base(httpClient, logger)
         { }
 
         protected override void SetIdentityInformations(ClaimsIdentity identity, string claimedID, IDictionary<string, string> attributeExchangeProperties)
         {
+            if (!Options.QueryProfile) return;
```

Comment:
is there any reason why we need this setting? Do these properties require special scopes to access?